bot/plugins/instance.py

```diff
@@ -488,11 +529,20 @@ async def execute(self) -> None:
             out.append("No runtime selected.")
 
         stdin = self.stdin or ""
-        formatted_stdin = "\n".join(
-            f"\x1b[1;33mIN:\x1b[0m {line}" for line in stdin.splitlines()
+        formatted_stdin = "\n".join(f"{line}" for line in stdin.splitlines())
```

Comment:
`"\n".join("f{line}" for line in stdin.splitlines())` is the same as just using `stdin`. This was just here to add the formatting before it.
```suggestion
```

Reply:
Good catch. I removed the `formatted_comptime_args` and the `formatted_runtime_args` as well.